tests/routes/view.test.js

```diff
@@ -505,7 +505,7 @@ describe('Routes', () => {
 
   it('bulk matching route should call the case service to fetch case data', async () => {
     const response = await request(app).get('/B14LO/match/bulk/2020-01-01')
-    expect(caseService.getPagedCaseList).toHaveBeenCalledWith('B14LO', '2020-01-01')
+    expect(caseService.getPagedCaseList).toHaveBeenCalledWith('B14LO', '2020-01-01', null, null, 1, 20, false)
```

Comment:
pedantic of me sorry, maybe add `to fetch first 20` to the title. idk what hearingOutcomesEnabled: false means

`bulk matching route should call the case service to fetch first 20 cases data`